Makefile

```diff
@@ -69,7 +69,12 @@ plugin-image-push: ## push the plugin image to the local  K8s cluster
 
 .PHONY: go-build
 go-build: ## build the Go source
-	@CGO_ENABLED=0 go build -o velero-plugin-storj .
+	@CGO_ENABLED=0 go build -o velero-plugin-storj ./cmd
+
+.PHONY: test
+test: ## Run tests on source code
+	cd testsuite; go test -race -v -cover -coverprofile=.coverprofile ./...
+	@echo done
```

Comment:
Is it needed?

Reply:
I copy-pasted this target from other Makefiles we have. I guess it is a helpful prompt.